From ac448a58a87b36eceb3ca99e6520af79640f7f24 Mon Sep 17 00:00:00 2001
From: Jerome Forissier <jerome.forissier@linaro.org>
Date: Fri, 6 Jul 2018 10:32:53 +0200
Subject: [PATCH] Do not set -Werror by default

Do not use -Werror unless explicitly asked via "make CFG_WERROR=y".

Signed-off-by: Jerome Forissier <jerome.forissier@linaro.org>
Reviewed-by: Joakim Bech <joakim.bech@linaro.org>
---
 android_flags.mk | 5 ++++-
 flags.mk         | 5 ++++-
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/android_flags.mk b/android_flags.mk
index 78bb306..34e3085 100644
--- a/android_flags.mk
+++ b/android_flags.mk
@@ -9,7 +9,10 @@ CFLAGS          := -Wall -Wbad-function-cast -Wcast-align \
 		   -Wmissing-noreturn -Wmissing-prototypes -Wnested-externs \
 		   -Wpointer-arith -Wshadow -Wstrict-prototypes \
 		   -Wswitch-default \
-		   -Wwrite-strings -Werror
+		   -Wwrite-strings
+ifeq ($(CFG_WERROR),y)
+CFLAGS		+= -Werror
+endif
 CFLAGS          += -c -fPIC
 
 DEBUG       ?= 0
diff --git a/flags.mk b/flags.mk
index 71f3d18..ff293eb 100644
--- a/flags.mk
+++ b/flags.mk
@@ -14,7 +14,10 @@ CFLAGS          := -Wall -Wbad-function-cast -Wcast-align \
 		   -Wmissing-noreturn -Wmissing-prototypes -Wnested-externs \
 		   -Wpointer-arith -Wshadow -Wstrict-prototypes \
 		   -Wswitch-default -Wunsafe-loop-optimizations \
-		   -Wwrite-strings -Werror
+		   -Wwrite-strings
+ifeq ($(CFG_WERROR),y)
+CFLAGS		+= -Werror
+endif
 CFLAGS          += -c -fPIC
 
 DEBUG       ?= 0
-- 
2.7.4

